xbmc/utils/StringUtils.h

```diff
@@ -375,6 +375,13 @@ class StringUtils
    */
   static std::string FormatFileSize(uint64_t bytes);
 
+  /*! \brief Converts a cstring pointer (const char*) to a std::string.
+             In case nullptr is passed the result is an empty string.
+      \param cstr the const pointer to char
+      \return the resulting std::string or ""
+   */
+  static std::string StringOrEmpty(const char* cstr);
```

Comment:
"Converts a cstring pointer (const char*) to a std::string"

I would name the new helper function `CreateFromCString" as this is what it does.